libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -58,7 +59,11 @@ void SRV_Channel::output_ch(void)
                 // non-mapped rc passthrough
                 int16_t radio_in = c->get_radio_in();
                 if (passthrough_mapped) {
+                  if (rc().in_rc_failsafe() && ((1U<<passthrough_from) & SRV_Channels::get_rc_fs_mask())) {
```

Comment:
```suggestion
                  if (((1U<<passthrough_from) & uint32_t(SRV_Channels::rc_fs_mask)) && rc().in_rc_failsafe()) {
```

the disabled_mask is used like this lower-down in this function.

Reordering to avoid the function call to rc() if we don't need it.

Reply:
done